dev/a11y_assessments/test/snack_bar_test.dart

```diff
@@ -7,21 +7,25 @@ import 'package:flutter_test/flutter_test.dart';
 
 import 'test_utils.dart';
 
+import 'package:a11y_assessments/use_cases/snack_bar.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'test_utils.dart';
+
 void main() {
   testWidgets('snack bar can run', (WidgetTester tester) async {
     await pumpsUseCase(tester, SnackBarUseCase());
     const String snackBarText = 'Awesome Snackbar!';
     expect(find.text(snackBarText), findsNothing);
     await tester.tap(find.text('Show Snackbar'));
-    expect(find.text(snackBarText), findsNothing);
     await tester.pump();
     expect(find.text(snackBarText), findsOneWidget);
   });
 
   testWidgets('snack bar demo page has one h1 tag', (WidgetTester tester) async {
     await pumpsUseCase(tester, SnackBarUseCase());
-    final Finder findHeadingLevelOnes = find.bySemanticsLabel('SnackBar Demo');
     await tester.pumpAndSettle();
+    final Finder findHeadingLevelOnes = find.bySemanticsLabel('SnackBar Demo');
```

Comment:
Can you add a expect that the message is announced? See this file for example
https://github.com/flutter/flutter/blob/master/packages/flutter/test/semantics/semantics_service_test.dart